### PR TITLE
Quick Draw Tutorial: Added Dropout to `DropoutWrapper`

### PR DESCRIPTION
**Pull request by HanGuo97**
Currently, the dropout seems to be ignored in regular RNN given that it's not actually used in the `DropoutWrapper`:

```python
if params.dropout > 0.0:
      cells_fw = [tf.contrib.rnn.DropoutWrapper(cell) for cell in cells_fw]
      cells_bw = [tf.contrib.rnn.DropoutWrapper(cell) for cell in cells_bw]
```

### CODE REVIEW

**Comment by jaeyounkim** [comment]
Deprecated
